src/dodal/beamlines/i22.py

```diff
@@ -345,35 +400,42 @@ def panda4(
     )
 
 
+# d3 at p38
 def oav(
     wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
 ) -> AravisDetector:
     return device_instantiation(
-        NXSasOAV,
+        AravisDetector if IS_LAB else NXSasOAV,
         "oav",
-        "-DI-OAV-01:",
+        f"-DI-{'DCAM' if IS_LAB else 'OAV'}-01:",
         wait_for_connection,
         fake_with_ophyd_sim,
         drv_suffix="DET:",
         hdf_suffix="HDF5:",
-        metadata_holder=NXSasMetadataHolder(
-            x_pixel_size=(3.45e-3, "mm"),  # Double check this figure
-            y_pixel_size=(3.45e-3, "mm"),
-            description="AVT Mako G-507B",
-            distance=(-1.0, "m"),
+        metadata_holder=None
+        if IS_LAB
+        else NXSasMetadataHolder(
+            x_pixel_size=(1.72e-1, "mm"),
+            y_pixel_size=(1.72e-1, "mm"),
+            description="Dectris Pilatus3 2M",
+            type="Photon Counting Hybrid Pixel",
+            sensor_material="silicon",
+            sensor_thickness=(0.45, "mm"),
+            distance=(4711.833684146172, "mm"),
```

Comment:
Copy paste error. This is now wrong where it was correct before.

As above re: laying out the devices, again with the rationale about the metadata, although this is one case wehre most of the fields will be the same, the distance will be different once it's measured.

Reply:
done